mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -630,47 +650,87 @@ impl JsonRpcClient for NativeClientImpl {
     fn client_info(&self) -> String { UtxoJsonRpcClientInfo::client_info(self) }
 
     #[cfg(target_arch = "wasm32")]
-    fn transport(&self, _request: JsonRpcRequestEnum) -> JsonRpcResponseFut {
-        Box::new(futures01::future::err(ERRL!(
-            "'NativeClientImpl' must be used in native mode only"
+    fn transport(&self, request: JsonRpcRequestEnum) -> JsonRpcResponseFut {
+        Box::new(futures01::future::err(JsonRpcError::new(
+            &self.coin_name(),
+            request,
+            JsonRpcErrorType::Transport("'NativeClientImpl' must be used in native mode only".to_string()),
```

Comment:
We can add an `JsonRpcErrorType::Internal` variant for this error. It should be also useful in mapping `SlurpError`.

Reply:
good idea!